app/main/views/index.py

```diff
@@ -189,12 +207,76 @@ def trial_mode_new():
     )
 
 
+@main.route("/best-practices")
+@user_is_logged_in
+def best_practices():
+    return render_template(
+        "views/best-practices/best-practices.html",
+        navigation_links=best_practices_nav(),
+    )
+
+
+@main.route("/best-practices/clear-goals")
+@user_is_logged_in
+def clear_goals():
+    return render_template(
+        "views/best-practices/clear-goals.html",
+        navigation_links=best_practices_nav(),
+    )
+
+
+@main.route("/best-practices/rules-and-regulations")
+@user_is_logged_in
+def rules_and_regulations():
+    return render_template(
+        "views/best-practices/rules-and-regulations.html",
+        navigation_links=best_practices_nav(),
+    )
+
+
+@main.route("/best-practices/establish-trust")
+@user_is_logged_in
+def establish_trust():
+    return render_template(
+        "views/best-practices/establish-trust.html",
+        navigation_links=best_practices_nav(),
+    )
+
+
+@main.route("/best-practices/write-for-action")
+@user_is_logged_in
+def write_for_action():
+    return render_template(
+        "views/best-practices/write-for-action.html",
+        navigation_links=best_practices_nav(),
+    )
+
+
+@main.route("/best-practices/multiple-languages")
+@user_is_logged_in
+def multiple_languages():
+    return render_template(
+        "views/best-practices/multiple-languages.html",
+        navigation_links=best_practices_nav(),
+    )
+
+
+@main.route("/best-practices/benchmark-performance")
+@user_is_logged_in
+def benchmark_performance():
+    return render_template(
+        "views/best-practices/benchmark-performance.html",
+        navigation_links=best_practices_nav(),
+    )
+
+
 @main.route("/using-notify/guidance")
 @user_is_logged_in
 def guidance_index():
     return render_template(
         "views/guidance/index.html",
         navigation_links=using_notify_nav(),
+        feature_best_practices_enabled=feature_best_practices_enabled
```

Comment:
Also given the above, this would have to change as well to this:

```suggestion
        feature_best_practices_enabled=current_app.config["FEATURE_BEST_PRACTICES_ENABLED"]
```

And you'll still get the value available in the templates!